gsuite_activityevent_rules/gsuite_login_type.py

```diff
@@ -14,6 +14,12 @@ def rule(event):
     if event.get("type") != "login":
         return False
 
+    if (
+        event.get("type") == "login"
+        and deep_get(event, "id", "applicationName") == "saml"
```

Comment:
Good catch on this!

The way this rule is written is to not assume what login types (or in this case applications it seems) an organization does or does not trust, and so users can customize the `APPROVED_LOGIN_TYPES` set to change what they do or don't approve of. Writing the check like this breaks that model.

What about this, what if we add a new set above called `APPROVED_APPLICATION_NAMES` that just has `"saml"` there, then we update the below check to:

```python3
    if (
        event.get("type") == "login"
        and event.get("name") != "logout"
        and deep_get(event, "parameters", "login_type") not in APPROVED_LOGIN_TYPES
        and deep_get(event, "id", "applicationName") not in APPROVED_APPLICATION_NAMES
    ):
```

Do you think that would cover this?